bundles/org.openhab.binding.siemensrds/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -184,20 +214,32 @@
 		<label>Occupancy Mode Present</label>
 		<description>The thermostat is in Present Occupancy Mode (Off = Away Mode)</description>
 		<category>presence</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Presence</tag>
```

Comment:
You cannot control the `Presence`, you control the `mode`.

(if you turn this on, one is not teleported into the room ;-) )

Reply:
Ok. I will create a new property `HomeAway`

Reply:
@lsiepel I am reverting this to Presence

Reply:
Was not asking you to change the name of the Property. `HomeAway` or `Presence` is just a different name for the same. This channel controls a `Mode` It is already in the label and description. It does not  control ones `Presence`.

Without knowing the binding, but now or with later api version it might have values like:  Home, Away, Off, On, Manual, Auto, GeoFencing, Aproaching, Leaving. Or whatever the modes might be named.

Reply:
https://github.com/openhab/openhab-core/pull/4708#issuecomment-2796294502

Reply:
> Nope. It is binary on/off i.e. home / away aka present / not present

Missing my point there. The longer i think about it the more i'm convinced this is wrong. Presence is not something you can control, it would be god like.

Let me try it once more:

In the case of this Channel, you control the 'Mode' the heater is in. You can be physically present in the room, but still manually set the presence **mode** to 'Away'. So it does not control the Presence. You are not teleported in and out of the house. It controls the mode and maybe a bit confusing, the mode values can be home or away (but that might change in future firmwares etc)

Reply:
I give up.